recipes/boost/all/test_package/CMakeLists.txt

```
@@ -116,6 +116,14 @@ if(NOT HEADER_ONLY)
         target_include_directories(numpy_exe PRIVATE ${PYTHON_INCLUDE_DIRS})
         target_link_libraries(numpy_exe PRIVATE Boost::numpy${PYTHON_COMPONENT_SUFFIX} ${PYTHON_LIBRARIES})
     endif()
+
+    if(WITH_URL)
+        find_package(Boost COMPONENTS url REQUIRED)
+        add_executable(url_exe url.cpp)
+        target_link_libraries(url_exe PRIVATE Boost::url)
+        set_property(TARGET url_exe PROPERTY CXX_STANDARD 11)
```

once rebasted on top of the most recent revision of the upstream `master` branch, this now needs:

```suggestion
        set_property(TARGET url_exe PROPERTY CXX_STANDARD 11)
        add_test(NAME boost_url COMMAND url_exe)
```